com.unity.multiplayer.mlapi/Runtime/Security/BigInteger.cs

```diff
@@ -1,2920 +0,0 @@
-/*
-   BigInteger Class Version 1.03
-   
-   Copyright (c) 2002 Chew Keong TAN All rights reserved.
-   
-   Permission is hereby granted, free of charge, to any person obtaining a copy of
-   this software and associated documentation files (the "Software"), to deal in
-   the Software without restriction, including without limitation the rights to
-   use, copy, modify, merge, publish, distribute, and/or sell copies of the
-   Software, and to permit persons to whom the Software is furnished to do so,
-   provided that the above copyright notice(s) and this permission notice appear in
-   all copies of the Software and that both the above copyright notice(s) and this
-   permission notice appear in supporting documentation.
-   
-   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
-   FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT OF THIRD PARTY RIGHTS. IN NO EVENT
-   SHALL THE COPYRIGHT HOLDER OR HOLDERS INCLUDED IN THIS NOTICE BE LIABLE FOR ANY
-   CLAIM, OR ANY SPECIAL INDIRECT OR CONSEQUENTIAL DAMAGES, OR ANY DAMAGES
-   WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF
-   CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION
-   WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
-*/
-#if !DISABLE_CRYPTOGRAPHY
-using System;
-using System.Security.Cryptography;
-
-namespace MLAPI.Security
-{
-    /// <summary>
-    /// This is a BigInteger class. Holds integer that is more than 64-bit (long).
-    /// </summary>
-    /// <remarks>
-    /// This class contains overloaded arithmetic operators(+, -, *, /, %), bitwise operators(&amp;, |) and other
-    /// operations that can be done with normal integer. It also contains implementation of various prime test.
-    /// This class also contains methods dealing with cryptography such as generating prime number, generating
-    /// a coprime number.
-    /// </remarks>
-
-    internal class BigInteger
```

Comment:
indeed, we removed this too :) @TwoTenPvP

Reply:
👍🏻 Some wierdness on my end when reviewing :P